CMake Error at while installing amr-wind: string sub-command STRIP requires two arguments.

I posted such an issue at the amr-wind project, but then were told that the issue comes from  the original AMReX. That seems to be true.

I was following instruction from here:
https://exawind.github.io/amr-wind/user/build.html

Running command
$ cmake -DAMR_WIND_ENABLE_TESTS:BOOL=ON ../

has led to the following error:
CMake Error at submods/amrex/Tools/CMake/AMReXUtils.cmake:20 (string):
string sub-command STRIP requires two arguments.

Investigation of the mentioned file has shown the difference between strings containing STRIP command:

![image](https://user-images.githubusercontent.com/46995837/129211436-1acd936c-6f22-4fa0-a6ef-be6ab9a1ecee.png)


string( STRIP ${_tmp} _tmp ) at line 20
VS
string(STRIP "${_tmp}" _tmp ) at line 32

Solution:
I followed a tip described here:
https://stackoverflow.com/questions/26585513/cmake-error-with-string-sub-command-strip-requires-two-arguments

adding the same quotas to line 20 seemed to solve the problem.

Original amr-wind issue: https://github.com/Exawind/amr-wind/issues/472

--- a/Tools/CMake/AMReXUtils.cmake
+++ b/Tools/CMake/AMReXUtils.cmake
@@ -17,7 +17,7 @@ function (get_amrex_version)
       execute_process ( COMMAND git describe --abbrev=12 --dirty --always --tags
          WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
          OUTPUT_VARIABLE _tmp )
-      string( STRIP ${_tmp} _tmp )
+      string( STRIP "${_tmp}" _tmp )
       # filter invalid descriptions in shallow git clones
       if (NOT _tmp MATCHES "^([0-9]+)\\.([0-9]+)(\\.([0-9]+))*(-.*)*$")
          set( _tmp "")

